modules/go-health-monitoring.adoc

```diff
@@ -0,0 +1,19 @@
+[id="health-information-resources_{context}"]
+= Checking health information
+
+.Prerequisites
+
+* {gitops-title} Operator installed from *OperatorHub*.
+* Argo CD applications are in sync.
+
+.Procedure
+
+* The Operator will install the backend service in `openshift-gitops` namespace. Click *Environments* under the *Developer* perspective. 
+
+*  The *Environments* page shows the list of applications along with their *Environment status*.
```

Comment:
This is not a step that the user has to take, but an explanation. Also there is an extra space after the * bullet point.

Reply:
If you feel that this explanation is necessary here, nest it under the point before, to make sure it is not misunderstood by the user as a step they have to perform.